package.json

```diff
@@ -10,18 +10,20 @@
   "bin": {
     "vsr": "./bin/vsr"
   },
+  "type": "module",
   "scripts": {
-    "setup": "wrangler d1 execute vsr_local_database --command \"CREATE TABLE IF NOT EXISTS packages ('name' TEXT PRIMARY KEY, tags JSON); CREATE TABLE IF NOT EXISTS tokens (token TEXT PRIMARY KEY, uuid TEXT, scope JSON); CREATE TABLE IF NOT EXISTS versions (spec TEXT PRIMARY KEY, manifest JSON, published_at TEXT); INSERT OR REPLACE INTO tokens (token, uuid, scope) VALUES ('xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'admin', '[ { \\\"values\\\": [\\\"*\\\"], \\\"types\\\": { \\\"pkg\\\": { \\\"read\\\": true, \\\"write\\\": true } } }, { \\\"values\\\": [\\\"*\\\"], \\\"types\\\": { \\\"user\\\": { \\\"read\\\": true, \\\"write\\\": true } } } ]');\" --local --persist-to local-store --experimental-json-config",
-    "drop": "wrangler d1 execute vsr_local_database --command 'DROP TABLE IF EXISTS packages;DROP TABLE IF EXISTS tokens;DROP TABLE IF EXISTS versions;' --local --persist-to local-store --experimental-json-config && rm -rf local-store && rm -rf .wrangler",
-    "migrate": "wrangler d1 migrations apply vsr_local_database  --local --persist-to local-store --experimental-json-config",
-    "dist": "wrangler dev ./dist/index.js --local --persist-to local-store --experimental-json-config",
-    "dev": "wrangler dev --local --persist-to local-store --experimental-json-config",
-    "bundle": "wrangler deploy --dry-run --outdir dist",
-    "deploy": "wrangler deploy",
-    "build": "NODE_NO_WARNINGS=1 node ./scripts/build.mjs",
+    "db:setup": "dmno run -- node scripts/db.js init",
+    "db:drop": "dmno run -- node scripts/db.js drop",
+    "db:migrate": "dmno run -- node scripts/db.js migrate",
+    "dev": "dwrangler dev --local --persist-to local-store",
+    "bundle": "dwrangler deploy --dry-run --outdir dist",
+    "dev:bundled": "wrangler dev ./dist/index.js --local --persist-to local-store",
```

Comment:
ok so I believe using `npx` does not install devDependencies - so  if you still want the vsr bin command to just run the thing without installing any deps, then including the dist folder and running via `wrangler` (instead of the wrapper) makes sense.

Reply:
Ahhh, ya. We should keep this for now then. We don't publish `vsr` to the npm registry (_yet_). I might in the future but it's currently installed/executed as a git repo (ie. running `npx vltpkg/vsr` is the same as `github:vltpkg/vsr`)